lib/BugReporting.dart

```diff
@@ -22,6 +22,12 @@ enum ReportType {
   OTHER
 }
 
+enum ExtendedBugReportMode {
```

Comment:
According to Dart's style guide, enums should use lowerCamelCase. https://www.dartlang.org/guides/language/effective-dart/style#prefer-using-lowercamelcase-for-constant-names